extract_otp_secret_keys.py

```diff
@@ -85,7 +88,9 @@ def print_qr(data):
     if verbose: print(payload)
 
     # pylint: disable=no-member
+    i = int(0)
```

Comment:
Is int() necessary?

Reply:
It's not, I don't remember why I put that there. Taken out now.